### ISSUE TITLE
Auto Creating tickets by email

### ISSUE THREAD

**Issue by nakuld21** [osTicket/osTicket#364]
Hi,

I really hope someone answers this query of mine.

I set up osticket 1.8.0.1 and email is working fine.
Email working means, after adding my host and smtp port settings, OsTicket system is able to send email/alerts to clients and the admin after tickets are created.

first, I'm confused between piping and polling.

But now, I also want the functionality where the system should automatically create a ticket if my client sends an email to : support@mydomain.com
it should automatically create a ticket in the system and this client should also get a usual reply with the ticket number

so, basically I want both methods: ticket creation via web and ticket creation by sending email to support@mydomain.com

Kindly help me at the earliest.

Thanks a lot

Rick

**Comment by tangchuanzhou** [comment]
for linux crontab, I think we could use command "whereis php" then get the /path/to/php,
for cron.php, we could use find to find where cron.php locates then get the /path/to/osticket/api/cron.php
then we could set the crontab job

**Comment by ntozier** [comment]
@tangchuanzhou 
This doesn't appear to be on topic.
PHP already does this typing in "php -i" at the command line.
Or perhaps you would be happier typing "which php"?

**Comment by arunarunit** [comment]
is it possible to convert emails into tickets in corephp, without installing osticket
-------------------------------------
 <p  style="text-align:center"><p> ADs: <a href="https://villagehosting.in/hosting/linux-shared-hosting" target="_blank">Shared Hosting</a>||<a href="https://villagehosting.in/hosting/linux-reseller-hosting" target="_blank">Reseller Hosting </a>||<a href="https://villagehosting.in/hosting/linux-vps-hosting/" target="_blank">VPS virtual Private servers </a>||<a href="https://villagehosting.in/hosting/dedicated-server" target="_blank">Dedicated Sever</a> || <a href="https://allbills.in/" target="_blank">Recharge API</a> || <a href="https://knuthers.com/" target="_blank">SSL Certificates</a> </p>

**Comment by JediKev** [comment]
@arunarunit 
>without installing osticket

This repo is for osTicket inquiries only. Your question would be better suited for PHP's repo/forum.

**Comment by andrelindner** [comment]
> Hi,
> 
> I really hope someone answers this query of mine.
> 
> I set up osticket 1.8.0.1 and email is working fine.
> Email working means, after adding my host and smtp port settings, OsTicket system is able to send email/alerts to clients and the admin after tickets are created.
> 
> first, I'm confused between piping and polling.
> 
> But now, I also want the functionality where the system should automatically create a ticket if my client sends an email to : [support@mydomain.com](mailto:support@mydomain.com)
> it should automatically create a ticket in the system and this client should also get a usual reply with the ticket number
> 
> so, basically I want both methods: ticket creation via web and ticket creation by sending email to [support@mydomain.com](mailto:support@mydomain.com)
> 
> Kindly help me at the earliest.
> 
> Thanks a lot
> 
> Rick

still an issue? if not, please close this issue.
